src/glass/src/app/pages/deployment-page/deployment-page.component.html

```diff
@@ -106,7 +106,7 @@
           {{ 'Configure CephFS service' | translate }}
           <mat-icon class="glass-color-accent"
                     svgIcon="mdi:plus-circle-outline"
-                    (click)="openCephfsDialog()">
+                    (click)="openFileServiceDialog('cephfs')">
```

Comment:
Something to keep in mind for the future, it would be nice if we could get rid of adding NFS and CephFS services individually and just had the possibility of adding a "FileService"; especially given the possibility of choosing the backing store for the service, it feels a bit moot still supporting adding services individually.